homeassistant/components/media_player/mpd.py

```diff
@@ -176,9 +177,13 @@ def media_title(self):
         """Return the title of current playing media."""
         name = self._currentsong.get('name', None)
         title = self._currentsong.get('title', None)
+        file_name = self._currentsong.get('file', None)
 
         if name is None and title is None:
-            return "None"
+            if file_name is None:
+                return "None"
+            else:
+                return os.path.basename(file_name)
```

Comment:
That is a property. You should not access to filesystem there. You can use Pathlib PurePath to handle that without os filesystem level